linkerd/app/outbound/src/lib.rs

```diff
@@ -49,9 +49,27 @@ const EWMA_DECAY: Duration = Duration::from_secs(10);
 
 #[derive(Clone, Debug)]
 pub struct Config {
-    pub proxy: ProxyConfig,
     pub allow_discovery: AddrMatch,
 
+    pub proxy: ProxyConfig,
+
+    /// Configures the duration the proxy will retain idle stacks (with no
+    /// active connections) for an outbound address. When an idle stack is
+    /// dropped, all cached service discovery information is dropped.
+    pub discovery_idle_timeout: Duration,
+
+    /// Configures how connections are buffered *for each outbound address*.
```

Comment:
nit:
```suggestion
    /// Configures how many connections are buffered *for each outbound address*.
```

Reply:
It's not just how many, it's also for how long. So I think the original is correct... it configures the buffering behavior.

Reply:
mmm, okay, yeah, nevermind.